cypress/e2e/hub/namespaces.cy.ts

```diff
@@ -26,6 +28,23 @@ describe('Namespaces', () => {
     cy.url().should('not.include', `/namespaces/${namespaceName}/details`);
   });
 
+  it('should show the correct URL when clicking on the CLI configuration tab', () => {
+    cy.navigateTo('hub', Namespaces.url);
+    const namespaceName = `test_namespace_${randomString(5, undefined, { isLowercase: true })}`;
+    cy.get('[data-cy="create-namespace"]').should('be.visible').click();
+    cy.url().should('include', Namespaces.urlCreate);
+    cy.get('[data-cy="name"]').type(namespaceName);
+    cy.get('[data-cy="company"]').type('test company');
+    cy.get('[data-cy="Submit"]').click();
+    cy.url().should('include', `/namespaces/${namespaceName}/details`);
+    cy.get('*[aria-controls^="pf-tab-section-hub-namespace-cli"]').should(
```

Comment:
It would worth adding a data-cy locator?

Reply:
I'm happy to add the data-cy locator, but finding the place on the UI to add this escapes me. Do you have a suggestion?